### PR TITLE
update with the added CLI interactions category

### PR DESCRIPTION
**Pull request by UnleashedMind**
…support

### CODE REVIEW

File: js/interactions.md [hunk 1]
```diff
@@ -12,13 +12,50 @@ Ensure you have [installed and configured the Amplify CLI and library]({%if jeky
...
+$ amplify add interactions
+```
+
+The CLI will lead you through the steps to specify the chatbot to be created. 
+
+You can choose to start from a sample chatbot or start from scratch.  If you choose to start from scratch, the CLI will prompt you with a series of questions to set the intents and slots for the chatbot. 
+
+You are allowed to run the `amplify add interactions` command multiple times to add multiple chatbots into your project.
+
+{The Interactions category utilizes the Authentication category behind the scenes to authorize your app to send analytics events.}
```
Comment (undefobj) [inline js/interactions.md]: Something is buggy here - "{The Interactions category utilizes the Authentication category behind the scenes to authorize your app to send analytics events.} {: .callout .callout--info}"

File: js/interactions.md [hunk 2]
```diff
@@ -12,13 +12,50 @@ Ensure you have [installed and configured the Amplify CLI and library]({%if jeky
...
+
+Import and load the configuration file in your app. It's recommended you add the Amplify configuration step to your app's root entry point. For example `App.js` in React or `main.ts` in Angular.
+
+```js
+import Amplify, { Interactions } from 'aws-amplify';
+import aws_exports from './aws-exports';
+
+Amplify.configure(aws_exports);
+```
+
```
Comment (undefobj) [inline js/interactions.md]: Can we add a link to jump straight to the "Working with the Interactions API" section? Like "Click HERE for usage in your app"

File: js/interactions.md [hunk 3]
```diff
@@ -12,13 +12,50 @@ Ensure you have [installed and configured the Amplify CLI and library]({%if jeky
...
+### Manual Setup
+
+#### Create your Chatbot
 
 You can create Amazon Lex chatbox in Amazon Lex console. To create your bot, follow the steps shown in [Amazon Lex Developer Guide](https://docs.aws.amazon.com/lex/latest/dg/getting-started.html).
  
 ![Interactions]({%if jekyll.environment == 'production'%}{{site.amplify.docs_baseurl}}{%endif%}/media/images/interactions_lex_console_edit_bot.jpg){: class="screencap" style="max-height:600px;"}
```
Comment (undefobj) [inline js/interactions.md]: This is another bug - Interactions{: class="screencap" style="max-height:600px;"

**Comment by mlabieniec** [comment]
Ensure the links are correct, they have changed in the new repo. For example There is no /media/quick_start it is now /js/start

**Comment by UnleashedMind** [comment]
I made the changes that Richard requested. 
I checked the "build errors", I think those are jekyll build directives, it points to the production hosting url, and I just checked, they are actually correct.

**Pull request merged**